gulp-tasks/test.js

```diff
@@ -368,7 +368,7 @@ function getFiles() {
     if (IS_TRAVIS) {
       cmd += '$(git merge-base FETCH_HEAD master) FETCH_HEAD';
     } else {
-      cmd += '$(git merge-base master HEAD)';
+      cmd += '$(git merge-base origin/master HEAD)';
```

Comment:
@petele I had to change to this so the comparison of local files would test against remote master not the current local master.

Reply:
Did you test for local changes as well as for changes on Travis?